examples/custom-theme/README.md

```diff
@@ -0,0 +1,29 @@
+# Demo App
```

Comment:
You might want to update this doc
Also add an entry in `examples/README.md`

Reply:
done